scout/server/config.py

```diff
@@ -105,7 +105,7 @@
 # FEATURE FLAGS
 SHOW_CAUSATIVES = True
 SHOW_OBSERVED_VARIANT_ARCHIVE = True
-HIDE_ALAMUT_LINK = True
+HIDE_ALAMUT_LINK = False
```

Comment:
Note to self: revert before merging. ☺️

Reply:
We would have noticed soon enough, but yes, good idea!